### PR TITLE
Update Pin Xian PPP

### LINKED ISSUE #296: Update PPP for Pin Xian

### PR DESCRIPTION
**Pull request by hopinxian**
Close #296

### CODE REVIEW

File: docs/team/hopinxian.md [hunk 1]
```diff
@@ -5,18 +5,103 @@ title: Ho Pin Xian's Project Portfolio Page
 
 ## Project: VirusTracker
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+**VirusTracker is a desktop application to generate Covid-19 related statistics.**
+While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+VirusTracker helps users manage lists of people, locations and visits relevant to the pandemic.
```
Comment (hhdqirui) [inline docs/team/hopinxian.md:10]: I was informed that we will all have the same introduction for VirusTracker for this part? Can refer to Siang Ern's and Shulong's PPP.

**Review by hhdqirui — CHANGES_REQUESTED** [review]
After fixing the part that was commented, it is ready to merge.

File: docs/team/hopinxian.md (hunk 1 above)
Comment (hopinxian) [reply docs/team/hopinxian.md]: Done

**Review by hhdqirui — APPROVED** [review]
LGTM

**Pull request merged**